partials/header.hbs

```diff
@@ -1,113 +1,23 @@
-{{!--
-This header template is shared across all the pages.
---}}
-
-<header class="m-header {{#if background}}with-picture{{/if}} js-header">
-  <div class="m-mobile-topbar" data-aos="fade-down">
-    <button class="m-icon-button in-mobile-topbar js-open-menu" aria-label="Open menu">
-      <span class="icon-menu"></span>
-    </button>
-    {{#if @site.logo}}
-      <a href="{{@site.url}}" class="m-logo in-mobile-topbar">
-        <img src="{{@site.logo}}" alt="{{@site.title}}">
-      </a>
-    {{else}}
-      <a href="{{@site.url}}" class="m-site-name in-mobile-topbar">
-        {{@site.title}}
-      </a>
-    {{/if}}
-    <button class="m-icon-button in-mobile-topbar js-open-search" aria-label="Open search">
-      <span class="icon-search"></span>
-    </button>
-  </div>
-
-  <div class="m-menu js-menu">
-    <button class="m-icon-button outlined as-close-menu js-close-menu" aria-label="Close menu">
-      <span class="icon-close"></span>
-    </button>
-    <div class="m-menu__main" data-aos="fade-down">
-      <div class="l-wrapper">
-        <div class="m-nav">
-          <nav class="m-nav__left">
-            <ul role="submenu">
-              {{#if @site.logo}}
-                <li class="only-desktop" role="menuitem">
-                  <a href="{{@site.url}}" class="m-logo">
-                    <img src="{{@site.logo}}" alt="{{@site.title}}">
-                  </a>
-                </li>
-                {{else}}
-                <li class="only-desktop" role="menuitem">
-                  <a href="{{@site.url}}" class="m-site-name in-desktop-menu">
-                    {{@site.title}}
-                  </a>
-                </li>
-              {{/if}}
-              {{#if @site.navigation}}
+<header class="home-header">
+<nav class="site-nav">
+	<a href="{{@site.url}}" class="nav-left">
```

Comment:
This element basically overlaps with the hamburger menu on small screens so you're likely to accidentally click the link to home rather than the hamburger menu. 

<img width="1439" alt="Screen Shot 2020-05-22 at 10 10 05 AM" src="https://user-images.githubusercontent.com/45807756/82693487-28433a80-9c51-11ea-8701-ce263f4d5fea.png">